Kickstarter-iOS/DataSources/CommentsDataSource.swift

```diff
@@ -0,0 +1,28 @@
+import KsApi
+import Library
+import UIKit
+
+internal final class CommentsDataSource: ValueCellDataSource {
```

Comment:
Just a note that I think @moderateepheezy might also be working on a data source too in another PR so there may be conflicts that we'd need to resolve.

Reply:
I noted the things we might conflict on in his [PR](https://github.com/kickstarter/ios-oss/pull/1480). But I'll wait until he merges his into master and then I'll fix conflicts. Just cuz I think in scrum (at least on the Android side) this ticket is like the last one going in.

Reply:
Spoke to @moderateepheezy about this yesterday, we agreed that `DemoComment` would become`Comment` and any fields that are currently missing in this PR, we would add later once the UI for Comment Card is merged in.